### PR TITLE
Intellectual property

### PR DESCRIPTION
**Pull request by smaciejs**

### CODE REVIEW

**Comment by mkurek** [comment]
@smaciejs could you first try to rename model `TradeMarksLinkedDomains` to `TradeMarksLinkedDomain`? Django should ask you if you renamed this model and should prepare proper migration automatically - https://github.com/allegro/ralph/blob/981875c8bf1c26710d5d72de97ef0af2a9215d02/src/ralph/trade_marks/models.py#L110

**Comment by smaciejs** [comment]
@mkurek  ok, I'll chagne it

**Comment by mkurek** [comment]
migration 0001 was already merged - we should not modify it - there should be 0002 migration which just rename the model (and maybe some fields)

File: src/ralph/trade_marks/migrations/0002_auto_20181004_1417.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('domains', '0006_auto_20180725_1216'),
+        ('trade_marks', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```
Comment (mkurek) [inline src/ralph/trade_marks/migrations/0002_auto_20181004_1417.py]: didn't django ask about renaming existing model? :) we should use `RenameModel` here (and `RenameField` below as well )

https://docs.djangoproject.com/en/1.8/ref/migration-operations/#renamemodel
https://docs.djangoproject.com/en/1.8/ref/migration-operations/#renamefield

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/trade_marks/migrations/0002_auto_20181004_1417.py (hunk 1 above)
Comment (smaciejs) [reply src/ralph/trade_marks/migrations/0002_auto_20181004_1417.py]: It didn't ask me about it. I'll do this in a proper way

**Comment by mkurek** [comment]
@smaciejs the tests are not passing - shouldn't you rename some fields as well?

**Review by mkurek — APPROVED** [review]

**Review by MarekBleschke — APPROVED** [review]

**Pull request merged**